### PR TITLE
Tombstone plugin

### PR DESCRIPTION
**Pull request by timja**
https://github.com/jenkinsci/jenkins/pull/5072#issuecomment-769253304

@jglick @jtnord

### CODE REVIEW

**Comment by timja** [comment]
Need to re add the icon taglib for compatibility I think

File: src/main/resources/index.jelly [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
```
Comment (jtnord) [inline src/main/resources/index.jelly]:
```suggestion
Copyright (c) 2021, Tim Jacomb
```

**Review by jtnord — COMMENTED** [review]
Seems correct to me.  @aHenryJard can you check this resolves the issue you where seeing?

**Comment by jtnord** [comment]
> Need to re add the icon taglib for compatibility I think

So tbh I have lost where this library is and went...

The readme states

> A "shim" plugin that allows plugins make use of the core <l:icon> tag, while still remaining backward compatible with older versions of Jenkins core that do not have the icon tag.

We are well past that - and the l:icon tag is in core - so if the baseline of this plugin is the new core then the taglib exists in master (`layout:icon`) so I do not think we have to add anything back here.
that said it is under shim/layout/icon so the taglib URI for inclusion would be different, so you may be correct here :/

At the same time - if that was the case I do not understand why we have had so many releases of this library.

Maybe @daniel-beck can recall something from his near encyclopaedic (when it comes to Jenkins) brain.

**Comment by timja** [comment]
Bit of background here from what I could find out last time:
https://github.com/jenkinsci/jenkins/pull/5072#issuecomment-734924531

will double check the code icon tag it might not be needed

File: pom.xml [hunk 2]
```diff
@@ -28,61 +28,37 @@ THE SOFTWARE.
...
     <license>
       <name>MIT License</name>
-      <url>http://opensource.org/licenses/MIT</url>
+      <url>https://opensource.org/licenses/MIT</url>
     </license>
   </licenses>
 
-  <modules>
-    <module>icon-set</module>
-    <module>plugin</module>
```
Comment (jglick) [inline pom.xml:56]: To keep the diff here smaller, I would have suggested leaving the parent POM alone and just updating `plugin`/, but OK.

**Review by jglick — APPROVED** [review]
Looks right?

**Comment by timja** [comment]
History seems to be explained here:
https://github.com/jenkinsci/plugin-pom/pull/72

big lol at https://github.com/jenkinsci/plugin-pom/pull/72#issuecomment-335483236

**Comment by timja** [comment]
I've done some testing and searching around.

There's two usages of icon-shim in the jenkinsci organisation

ecutest and docker-commons (both should be updated)

neither of them particularly easy to test this on, so I wrote a quick test plugin.

Without the shim jelly taglib then the icon didn't load, it was silent though, no log messages or errors.

I added it back in for added 'compatibility'

**Pull request merged**